testing/marketplace/deployer/helm_tiller_onbuild/helm-dependency-build/chart/chartmuseum-marketplace/requirements.yaml

```diff
@@ -0,0 +1,7 @@
+dependencies:
+- name: marketplace-integration
+  version: 0.0.1
+  repository: https://cloud-marketplace-tools-charts.storage.googleapis.com/
+- name: chartmuseum
+  version: 1.6.0
```

Comment:
Similarly, we want to specify `1.6.x` here.

Reply:
Done!